lasagne/layers/merge.py

```diff
@@ -6,7 +6,8 @@
 __all__ = [
     "ConcatLayer",
     "concat",
-    "ElemwiseSumLayer",
+    "ElemwiseMergeLayer",
+    "ElemwiseSumLayer"
```

Comment:
We usually have a comma also after the last line so you can add things in any position with a single inserted line instead of having to modify two lines. (Just so you know for the future. Not crucial now.)